pkg/config/config_test.go

```diff
@@ -389,3 +390,93 @@ func createOrdererStandardPolicies() map[string]*Policy {
 
 	return policies
 }
+
+func TestComputeUpdate(t *testing.T) {
+	gt := NewGomegaWithT(t)
+
+	value1Name := "foo"
+	value2Name := "bar"
+	base := common.Config{
+		ChannelGroup: &common.ConfigGroup{
```

Comment:
Let's reuse the cb import instead of having two names for the same import.

Reply:
🛠